### ISSUE TITLE
Correggere i termini che contengono la traduzione inglese "informations"

### ISSUE THREAD

**Issue by marco-deligios** [italia/daf-ontologie-vocabolari-controllati#184]
In English the word “information” is an uncountable noun.

**Comment by giorgialodi** [comment]
Dov'è questo obbrobrio?? In quale vocabolario nello specifico?

**Comment by marco-deligios** [comment]
transparency-titulus (2 occorrenze) ;)

**Comment by marco-deligios** [comment]
Non che **Pagamenti dell'amministrazione -> Payments administration** mi convinca....

**Comment by giorgialodi** [comment]
No hai ragione. Ok, correggo assolutamente queste cose che non vanno bene. Dammi un po' di tempo che sono in riunioni varie! Grazie Marco!

**Comment by marco-deligios** [comment]
Che dire di **Pianificazione e governo del territorio	-> Planning and government of the area**?

**Comment by giorgialodi** [comment]
sempre in transparency titulus? Così rivedo tutto l'inglese che facciamo prima.

**Comment by ioggstream** [comment]
Dovremmo aggiungere aspell nella CI

**Comment by marco-deligios** [comment]
A parte i typos il problema è che le traduzioni in inglese sembrano essere fatte con translator (ortograficamente corrette, ma incoerenti)

**Comment by giorgialodi** [comment]
Esatto, questa cosa non è da CI secondo me, perché va ragionata. Comunque, appena ho un attimo rivedo tutta la parte in inglese. @ioggstream

**Comment by ioggstream** [comment]
Sì, ma almeno gli errori grammaticali togliamoceli.:)

**Comment by giorgialodi** [comment]
Sì sì @ioggstream lo facciamo ma il tema qui è più generale.

**Comment by giorgialodi** [comment]
@marco-deligios @ioggstream  fatto! Con l'ultima PR che vedete! Appena qualcuno del team fa merge possiamo chiudere l'issue

**Comment by bfabio** [comment]
Ciao @giorgialodi, vedo ancora qualche occorrenza di "informations":

```console
❯ rg informations
VocabolariControllati/classifications-for-transparency/transparency-titulus/transparency-titulus.jsonld
3763:        "@value": "Other informations"

VocabolariControllati/classifications-for-transparency/transparency-titulus/transparency-titulus.ttl
1421:					 "Other informations"@en ,

VocabolariControllati/classifications-for-transparency/transparency-titulus/transparency-titulus.rdf
735:        <skos:prefLabel xml:lang="en">Other informations</skos:prefLabel>
```

**Comment by giorgialodi** [comment]
grazie @bfabio devo verificare. Ero convinta di averli sistemati

**Comment by giorgialodi** [comment]
@bfabio grazie per l'ulteriore check. In effetti è stato modificato nel file csv ma non nella versione RDF. Teniamo aperto, modifico e sottopongo una PR se per te va bene

**Comment by bfabio** [comment]
Perfetto, grazie @giorgialodi
